crates/holochain/src/core/workflow/publish_dht_ops_workflow/publish_query.rs

```diff
@@ -54,17 +57,16 @@ pub async fn get_ops_to_publish(
             AND
             (DhtOp.type != :store_entry OR Header.private_entry = 0)
             AND
-            DhtOp.last_publish_time IS NULL
+            (DhtOp.last_publish_time IS NULL OR DhtOp.last_publish_time <= :earliest_allowed_time)
```

Comment:
This is actually the `latest_allowed_time`, so either call it that, or I like to call these things "thresholds"

```suggestion
            (DhtOp.last_publish_time IS NULL OR DhtOp.last_publish_time <= :recency_threshold)
```